flocx_market/db/sqlalchemy/api.py

```diff
@@ -52,6 +52,11 @@ def offer_get_all_unexpired():
         .filter(models.Offer.status != 'expired').all()
 
 
+def offer_get_all_available():
```

Comment:
Let's have this take in a `status` argument instead of hardcoding `available`; we can call the method `offer_get_all_by_status`